package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@sketch7/signalr-orleans",
-  "version": "3.1.0",
+  "version": "3.2.0",
```

```suggestion
  "version": "3.1.0",
```
revert this, version will be added in the release branch